### DISCUSSION TITLE
LAI attri files

### DISCUSSION THREAD

**Discussion by Shuzunyun** [NASA-LIS/LISF#1101] [Q&A] [answered]
Dear all, 

I have found some attrib and pertattrib files about soil moisture, forcing and snow at https://github.com/NASA-LIS/LISF/tree/master/lis/configs/557WW-7.4-FOC/NRT_GLOBAL/attribs. But there are are no examples of these files about GLASS LAI. I am confused about how these parameters, e.g., varmin and varmax, are determined. Is there anyone else who can share these files or papers about how to determine these parameters? That will be a great appreciation.

Shu

**Comment by karsenau** [comment, marked as answer]
Dear @Shuzunyun:

Thank you for your interest in LIS.  For LAI DA perturbation related attribute files, you could look at some recent publications by our group that mention some of the information that you are looking for, e.g.:

Sujay Kumar et al. (2019):
https://journals.ametsoc.org/view/journals/hydr/20/7/jhm-d-18-0237_1.xml
Please see Section 2.b.

Wanshu Nie et al. (2022):
https://hess.copernicus.org/preprints/hess-2021-263/hess-2021-263.pdf
Please see section 2.2.2.

For max LAI values (varmin. = 0), you can check the following GLASS-related papers that list what max values you may see with that dataset:
http://www.glass.umd.edu/introduction.html
https://www.mdpi.com/2072-4292/8/4/351/htm
https://www.sciencedirect.com/science/article/pii/S0034425722000992

Typically, max LAI values can be as high as 10, depending on vegetation type.

**Reply by Shuzunyun** [reply]
Dear Karsenau,

Thanks for your reply. The links you mentioned above help me a lot.

Shu Zunyun

**Reply by karsenau** [reply]
You're welcome and glad you found them helpful.

**Reply by Shuzunyun** [reply]
Dear Karsenau,

There is a new question, and I would like to ask you for advise.

The question is what the meaning of standard deviation for perturbation is. The standard deviations applied for modeled LAI and observed LAI are 0.01 in Nie et al. (2022) and Mocko et al. (2021). If the 0.01 represents the errors of observation and model state, why the observation and model state are so accurate? If the observation is assumed to be accurate, whether there is need to do data assimilation. We can use the observation directly through interpolation. On the other hand, if the model state is accurate, whether there is need to be improved through DA.

This is just a discussion. It would be appreciated if you could reply.

Thanks!
Shu

**Reply by karsenau** [reply]
Hi @Shuzunyun:

These are good questions you have, and the standard error estimates (or "standard deviations", also covariance errors) can be estimated by validating the performance of the specific model states (e.g., the NoahMP dynamic scheme -- LAI state) or the MODIS (or other satellite-based) LAI estimates.  That is typically where the error estimates are derived from.

For more information, please check out:
Kumar et al. (2019):
https://journals.ametsoc.org/view/journals/hydr/20/7/jhm-d-18-0237_1.xml
and Xie et al. (2017):
https://www.sciencedirect.com/science/article/pii/S0168192317302149?via%3Dihub

If the error for the model states is low, then yes -- the observations may not have as much impact, whereas if the model error is higher and the observation error is very low, the observations should have much more impacts on the state updates or final analysis.

**Reply by Shuzunyun** [reply]
Dear Karsenau,

Thanks for helping me understand the hyper-parameters in data assimilation. Thanks again for your time.

Shu
